src/cli/setup_mode.rs

```diff
@@ -115,6 +115,10 @@ pub fn main() -> Result<()> {
         targets: &targets,
     };
 
+    if profile == "complete" {
+        warn!("downloading with complete profile isn't recommneded unless you are a developer of the rust language")
```

Comment:
This repeated typo suggests you might do well to have this as a constant string so there's no chance of mistakes in future.  You could put that usefully in `src/cli/common.rs` and then use it in both the locations.